FreeRTOS/iot-reference-arm-corstone3xx
# Add support for FreeRTOS TCP/IP stack

Add support for FreeRTOS TCP/IP stack

Description
-----------

This PR adds support for FreeRTOS TCP/IP stack and makes it the default network stack for the FRI.

Test Steps
-----------

Tested AWS IoT core end to end connectivity and OTA job.

Checklist:
----------



Related Issue
-----------
Depends on https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1009

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
